Adyen/UI/Form/Items/Phone Number/Phone Extension Picker/FormPhoneExtensionPickerItem.swift

```diff
@@ -19,6 +19,10 @@ extension PhoneExtension: FormPickable {
 @_spi(AdyenInternal)
 public final class FormPhoneExtensionPickerItem: FormPickerItem<PhoneExtension> {
     
+    internal var allowsSelection: Bool {
```

Comment:
This is outside of this PR scope, why is this called PhoneExtension? This is country code prefix, right?

Reply:
I think we are using the phone picker to get only one country Code here which is +61 and that's why we are removing the selection on it.

Reply:
I agree but I believe PhoneExtension is public - so changing it is breaking - we might want to rename it and create a typealias to keep the API stable - but yeah a bit out of scope here 👌

Reply:
Oh, disergard this specific line I commented on please. 
It's about generic naming of this `FormPhoneExtensionPickerItem`, to me it looks more like a `FormPhoneCountryCodePickerItem`

Reply:
I'm confused, what is out of scope and what is breaking? I don't see any breaking changes

Reply:
Naming is the issue, the name of the class is `FormPhoneExtensionPickerItem`. 
We cannot rename it to `FormPhoneCountryCodePickerItem` because it's `public` now.

And overall this discussion is outside of the scope of current PR, I raised this as `FormPhoneExtensionPickerItem` name doesn't provide correct context.